help/include/topic-notifications-intro.md

```diff
@@ -0,0 +1,23 @@
+To avoid receiving notifications messages you are not interested in, Zulip lets
+you mute topics and streams. Muting a stream automatically mutes all topics in
+that stream. You can also manually **mute** a topic in an unmuted stream, or
+**unmute** a topic in a muted stream.
+
+Muting has the following effects:
+
+- Messages in muted topics do not generate notifications (including [alert
+  word](/help/dm-mention-alert-notifications#alert-words) notifications), unless
+  you are [mentioned](/help/mention-a-user-or-group).
+- Messages in muted topics do not appear in the [**All messages**
+  view](/help/all-messages).
+- Muted topics appear in the [**Recent conversations**
+  view](/help/recent-conversations) only if the **Include muted** filter is
+  enabled.
+- Unread messages in muted topics do not contribute to stream unread counts.
+- Muted streams still appear in the left sidebar, but they are grayed out and
```

Comment:
Muted topics also have this happen when the stream is open; we might want to mention that?